training_toolbox/lpr/evaluate.py

```diff
@@ -0,0 +1,162 @@
+import random
+import numpy as np
+import tensorflow as tf
+import tensorflow.contrib.slim as slim
+import time
+from trainer import inference, LPRVocab, encode, decode_beams
+from lpr.toolbox.utils import accuracy, dataset_size
+from utils.helpers import load_module
+
+import os
```

Comment:
I think it is a strange order of imports. Do you have a chance to run `pylint` locally?

Reply:
Fixed.